bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -122,12 +125,9 @@ fn build_function_arn(account_id: &str, region: &str, function_name: &str) -> St
 #[allow(clippy::too_many_lines)]
 fn main() -> Result<()> {
     // First load the configuration
-    let lambda_directory = match env::var("LAMBDA_TASK_ROOT") {
-        Ok(val) => val,
-        Err(_) => "/var/task".to_string(),
-    };
-    let config = match config::get_config(Path::new(&lambda_directory)) {
-        Ok(config) => Arc::new(config),
+    let lambda_directory = env::var("LAMBDA_TASK_ROOT").unwrap_or_else(|_| "/var/task".to_string());
+    let env_config = match config::get_config(Path::new(&lambda_directory)) {
```

Comment:
This is not technically `env_config` since it includes also `YAML`, should we find a better name for this?

Reply:
Good point, I am thinking that not the config but only the secret will be returned before we merge this. I will rename it once I remove this extra config